src/migrations/20240624185931-unlock-tr-and-sessions.js

```diff
@@ -0,0 +1,27 @@
+const {
+  prepMigration,
+} = require('../lib/migration');
+
+module.exports = {
+  up: async (queryInterface) => queryInterface.sequelize.transaction(
+    async (transaction) => {
+      await prepMigration(queryInterface, transaction, __filename);
+      await queryInterface.sequelize.query(/* sql */`
+        UPDATE "SessionReportPilots"
+        SET data = jsonb_set(data, '{status}', '"In progress"', true)
+        WHERE "eventId" = 48;
```

Comment:
Just to confirm, this will set all session for that event to "In progress".

Reply:
Yes; that's how I interpreted the instruction to "Change both sessions in R10-TR-23-1045 from Complete to Not Completed"